### PR TITLE
link standalone content

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

link standalone content

**Preview:** https://hds-website-git-br-link-stand-hashicorp.vercel.app/components/link/standalone

See https://github.com/hashicorp/design-system/pull/933 for similar updates to Link::Inline

### :link: External links

Jira ticket: [HDS-1281](https://hashicorp.atlassian.net/browse/HDS-1281)

### CODE REVIEW

File: website/docs/components/link/standalone/partials/guidelines/guidelines.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+Use the Secondary variant for less important links, when the action color can't be used, or when creating a list of links.
+
+For example in an Install page, 'Learn more about Vault'
+
+<Hds::Link::Standalone @text="View clusters" @icon="external-link" @iconPosition="trailing" @color="secondary" @route="components" />
+
+### State
+#### Primary
+
+TBD Image here?
```
Comment (Dhaulagiri) [inline website/docs/components/link/standalone/partials/guidelines/guidelines.md:50]: @jorytindall  this is an example where we have this image in the existing guidelines but i'm not sure it is possible to recreate in code

File: website/docs/components/link/standalone/partials/accessibility/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+### Annotations
+
+Provide annotations alongside each design of the non-visual experience. This could look like:
+
+![Annotation of focus order of the link](/assets/components/link/standalone/link_standalone-accessibility-focus_order.png)
+
+!!! Info
+
+Animations or transitions will not take place if the user has `prefers-reduced-motion` enabled in their browser.
+!!!
```
Comment (MelSumner) [inline website/docs/components/link/standalone/partials/accessibility/accessibility.md:12]:
```suggestion
!!!

## Applicable WCAG Success Criteria

This section is for reference only. This component intends to conform to the following WCAG success criteria:

<Doc::WcagList @criteriaList={{array "1.3.1" "2.1.1" "1.3.3" "1.4.1" "2.4.7" }} />
```

File: website/docs/components/link/standalone/partials/guidelines/guidelines.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+### When not to use
+
+- Within a body of text, use a **Link (Inline)**
+- To trigger an action or event, like a form submission, use a [Button](/components/button/overview)
+
+### Size
+
+<Hds::Link::Standalone @size="large" @text="Large" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+<Hds::Link::Standalone @size="medium" @text="Medium" @icon="arrow-right" @iconPosition="trailing" @route="components" />
```
Comment (MelSumner) [inline website/docs/components/link/standalone/partials/guidelines/guidelines.md:16]:
```suggestion

<Hds::Link::Standalone @size="medium" @text="Medium" @icon="arrow-right" @iconPosition="trailing" @route="components" />

```
These need some breathing room, visually.

File: website/docs/components/link/standalone/partials/accessibility/accessibility.md (hunk 2 above)
Comment (Dhaulagiri) [reply website/docs/components/link/standalone/partials/accessibility/accessibility.md:12]: 🤦  aggg! thank you!

File: website/docs/components/link/standalone/partials/guidelines/guidelines.md (hunk 3 above)
Comment (Dhaulagiri) [reply website/docs/components/link/standalone/partials/guidelines/guidelines.md:16]: Agreed, but I kinda punted on that for this first pass. I'll commit this to see if it helps.

**Review by MelSumner — APPROVED** [review]

File: website/docs/components/link/standalone/partials/guidelines/guidelines.md (hunk 1 above)
Comment (Dhaulagiri) [reply website/docs/components/link/standalone/partials/guidelines/guidelines.md:52]: I think either way I should move these to the Specifications tab

Comment (Dhaulagiri) [reply website/docs/components/link/standalone/partials/guidelines/guidelines.md]: moved these to the correct place and added images

**Review by KristinLBradley — APPROVED** [review]

File: website/docs/components/link/standalone/partials/guidelines/guidelines.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+### Size
+
+<Hds::Link::Standalone @size="large" @text="Large" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+
+<Hds::Link::Standalone @size="medium" @text="Medium" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+
+<Hds::Link::Standalone @size="small" @text="Small" @icon="arrow-right" @iconPosition="trailing" @route="components" />
+
+Medium is the preferred size, but use a link size that best fits the UI.
```
Comment (jorytindall) [inline website/docs/components/link/standalone/partials/guidelines/guidelines.md:21]:
```suggestion
In most cases use a medium size, but use a link size that best fits the UI.
```

File: website/docs/components/link/standalone/partials/guidelines/guidelines.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+**Link (Standalone)** can commonly be seen within **button sets**.
+
+<Hds::ButtonSet>
+  <Hds::Link::Standalone @text="Previous" @icon="chevron-left" @iconPosition="leading" @color="primary" @route="components" />
+  <Hds::Button @text="Submit" @color="primary" />
+  <Hds::Button @text="Cancel" @color="secondary" />
+</Hds::ButtonSet>
+
+Often when included in a ButtonSet, Link (Standalone) is used for navigating between steps in a multi-page form (ie. directing the user back to the previous step). As such, they should be displayed as the left most button in the row.
```
Comment (jorytindall) [inline website/docs/components/link/standalone/partials/guidelines/guidelines.md:102]:
```suggestion
When included in a ButtonSet, Link (Standalone) is generally used for navigating between steps in a multi-page form (ie. directing the user back to the previous step) or for linking to external documentation. As such, they should be displayed as the right most button in the row.
```

This corresponds with the button alignment/ordering/grouping research.

**Review by jorytindall — APPROVED** [review]
✅ with a couple of non-blocking comments

File: website/docs/components/link/standalone/partials/code/how-to-use.md [hunk 6]
```diff
@@ -1,59 +1,71 @@
...
 If you pass a `@route` argument a `<a>` link will be generated using a `<LinkTo>` Ember component:
 
 ```handlebars
 <Hds::Link::Standalone @icon="collections" @text="Go to the index page" @route="my.page.route" @model="my.page.model" />
 ```
 
-**Important**: if the route is external to your current engine you have to pass also `@isRouteExternal={{true}}` to the component so that it will use `<LinkToExternal>` instead of a simple `<LinkTo>` for the `@route`. For more details see the [Hds::Interactive component](/utilities/interactive/)
+!!! Info 
+
+If the route is external to your current engine you have to pass also `@isRouteExternal={{true}}` to the component so that it will use `<LinkToExternal>` instead of a simple `<LinkTo>` for the `@route`. For more details see the [Hds::Interactive component](/utilities/interactive/)
```
[the remark this answers is not in the document]
Comment (Dhaulagiri) [reply website/docs/components/link/standalone/partials/code/how-to-use.md:65]: in this case "engine" is a known term (https://ember-engines.com/) that defines how projects like HCP divide their app into sub-applications.

**Pull request merged**